risc0/zkvm/src/exec/env.rs

```diff
@@ -151,6 +213,39 @@ impl<'a> ExecutorEnvBuilder<'a> {
     }
 
     /// Add initial input that can be read by the guest from stdin.
+    /// This step must occur before the executor environment is built.
```

Comment:
I don't understand this comment because structurally, it's not possible to `add_input` after the env has been built.

Reply:
Yeah this is trivially true, pulling it out.